treeherder/model/derived/jobs.py

```diff
@@ -431,6 +431,20 @@ def get_performance_series_summary(self, interval_seconds):
 
         return series_summary
 
+    def get_platforms(self, interval_seconds):
+        last_updated_limit = utils.get_now_timestamp() - interval_seconds
+        data = self.get_dhub().execute(
+            proc="jobs.selects.get_perf_series_properties",
+            placeholders=[last_updated_limit, interval_seconds],
+            debug_show=self.DEBUG,
+        )
+        platforms = []
+        for datum in data:
+            if datum['property'] == 'machine_platform':
+                if datum['value'] not in platforms:
+                    platforms.append(datum['value'])
+        return platforms
```

Comment:
You could express this more concisely by using set() and list comprehensions here:

https://docs.python.org/2/library/sets.html
https://docs.python.org/2/tutorial/datastructures.html#list-comprehensions

E.g.:

     platforms = set()
     for [datum for datum in data if datum['property'] == 'machine_platform']:
         platforms.add(datum['value'])